vmware-tanzu/tanzu-framework
# [release-0.11] Backport capv-secret-deletion-fix-to.0.11

### What this PR does / why we need it
This PR updates the CAPV version to `v1.0.3`. This new patch version contains the CAPV identity secret deletion fix. 
More details here - https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1486 
### Which issue(s) this PR fixes


Fixes #

### Describe testing done for PR
1. Create a workload cluster with TKG 1.4
2. Upgrade the workload cluster to TKG 1.5
3. Check the finalizers on the identity secret
4. 2 finalizers will be present - `{vsphere, identity}/infrastructure.cluster.x-k8s.io`
5. Delete the workload cluster
6. The secret should be successfully deleted(i.e., both the finalizers should be deleted). 



### Release note

```release-note
Fixed an issue where a finalizer on the VSphereIdentitySecret is not deleted in case of clusters upgraded from TKG 1.x to 1.5. 
```

### PR Checklist



      


### Additional information

#### Special notes for your reviewer
